src/scss/_typography.scss

```diff
@@ -51,6 +48,29 @@ body {
   font-style: italic;
 }
 
+/** Fonts **/
+@font-face {
+  font-family: Montserrat;
+  src: url('../fonts/Montserrat.woff') format('woff');
+  src: url('../fonts/Montserrat.woff2') format('woff2');
+}
+
+@mixin Montserrat {
+  font-family: Montserrat, sans-serif;
+}
+
+@font-face {
+  font-family: MontserratItalic;
+  font-style: italic;
+  src: url('../fonts/Montserrat-Italic.woff') format('woff');
+  src: url('../fonts/Montserrat-Italic.woff2') format('woff2');
+}
+
+@mixin MontserratItalic {
+  font-family: MontserratItalic, sans-serif;
+  font-style: italic;
+}
+
```

Comment:
The storybook is broken so I was not able to check but I think there would have to be more fonts downloaded to cover all of those different weights. 

I also think we should define all font-family names as "Montserrat" since this functionality better aligns with expectations. ie. Devs expect that to affect the font weight for font-style they would just have to add those properties. Defined this way the developer will have to change font names. This has already caused confusion in phillips-public.

Reply:
I fixed the storybook type tokens page, but I believe these are variable weight fonts so they work without loading the specific font weights

Reply:
See comment below.